index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My First Project</title>
+    <link rel="stylesheet" href="style.css">
+
```

Comment:
- [ **OPTIONAL** ] I suggest you remove this empty line to make your code clean and smart.

Reply:
I will also check on this. [@tresorsawasawa](https://github.com/tresorsawasawa) thank you.